tests/SwaggerServiceTest.php

```diff
@@ -683,6 +685,61 @@ public function testAddDataPostRequestWithObjectParams()
         $service->addData($request, $response);
     }
 
+    public function testAddDataWithNotExistsMethodOnController()
+    {
+        $this->mockDriverGetTmpData($this->getJsonFixture('tmp_data_get_user_request'));
+
+        $service = app(SwaggerService::class);
+
+        $request = $this->generateRequest(
+            type: 'get',
+            uri: 'users/{id}/assign-role/{role-id}',
+            data: [
+                'with' => ['role'],
+                'with_likes_count' => true,
+            ],
+            pathParams: [
+                'id' => 1,
+                'role-id' => 5,
+            ],
+            controllerMethod: 'notExists'
+        );
+
+        $response = $this->generateResponse('example_success_user_response.json', 200, [
+            'Content-type' => 'application/json',
+        ]);
+
+        $service->addData($request, $response);
+    }
+
+    public function testAddDataWithBindingInterface()
```

Comment:
Finally, this test is going to cover:
https://github.com/RonasIT/laravel-swagger/blob/688fe0b4146b8f9a71abb95e7971c47dcbbc1655/src/Traits/GetDependenciesTrait.php#L53-L55